### PR TITLE
Customer tests

### PR DESCRIPTION
**Pull request by ViktoriaMilkevich**

### CODE REVIEW

File: src/test/java/com/easyrest/tests/customer/WorkWithOrdersTests.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    public void goToSignInPage(){
+        signInPage = new SignInPage(driver);
+        signInFacade = new SignInFacade(driver);
+        signInPage.goToSignInPage();
+        signInFacade.signIn(clientEmail, clientPassword);
+    }
+
+    @Test
+    public void seeAllOrders(){
+        operationsWithOrderFacade = new OperationsWithOrderFacade(driver);
```
Comment (Zeicks) [inline src/test/java/com/easyrest/tests/customer/WorkWithOrdersTests.java:39]: If you initialize it in @BeforeTest will it work? To remove repetitions.

**Review by martadiachok — APPROVED** [review]

File: src/main/java/com/easyrest/pages/customerPanel/CurrentOrders.java [hunk 2]
```diff
@@ -44,8 +44,7 @@ public CurrentOrders seeOrderDetails() {
         return new CurrentOrders(driver);
     }
 
-    public CurrentOrders declineOrder() {
-        driver.findElement(decline).click();
-        return new CurrentOrders(driver);
+    public By getDeclineElement(){
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/pages/customerPanel/CurrentOrders.java:47]: the public modifier could break the page object encapsulation

**Review by Zeicks — APPROVED** [review]

File: src/main/java/com/easyrest/pages/customerPanel/CurrentOrders.java (hunk 2 above)
Comment (dmatsev) [reply src/main/java/com/easyrest/pages/customerPanel/CurrentOrders.java:47]: why do you need `By` as `public`?

**Review by dmatsev — COMMENTED** [review]
I left the comment.

**Review by dmatsev — APPROVED** [review]

**Pull request merged**